### DISCUSSION TITLE
Ai跑图出好货

### DISCUSSION THREAD

**Discussion by sb35386057** [share121/inter-knot#5491] [公告]
大概是几个月前的事,用ai关键词出有国旗的长安,得出此图,看着不错，所以用来当朋友圈头图，之后反复用重生的词生成也没生成出来这种的
![20240608112725](https://github.com/user-attachments/assets/7df7f79d-edb0-47c4-a55d-87bbbe02a9b6)

**Comment by sb35386057** [comment]
![Uploading 20240609234009.png…]()

**Comment by sb35386057** [comment]
![Uploading 20240610113400.png…]()

**Comment by cpu644** [comment]
这个整体构图没问题 主要小国旗不行 这种时候就得人工操作了

**Comment by Wax1234567** [comment]
建议谨慎一点，国旗不允许修改图案和比例哦，AI跑出来的东西，不好说
